e2e/packages/client-vanilla/src/mud/getNetworkConfig.ts

```diff
@@ -41,7 +41,7 @@ export async function getNetworkConfig(): Promise<NetworkConfig> {
       jsonRpcUrl: params.get("rpc") ?? chain.rpcUrls.default.http[0],
       wsRpcUrl: params.get("wsRpc") ?? chain.rpcUrls.default.webSocket?.[0],
     },
-    privateKey: getBurnerWallet().value,
+    privateKey: params.get("privateKey") ?? getBurnerWallet().value,
```

Comment:
this is so that we can provide a known private key in the tests (eg the deployer private key to be able to call `world.setRecord` from the client)